utils/utils.go

```diff
@@ -173,3 +181,31 @@ func HTTPGetAndParseBody(endpoint string) (string, error) {
 	}
 	return string(body), nil
 }
+
+func DoesCRDsExists(name string) bool {
+
+	var options client.Options
+	options.Scheme = runtime.NewScheme()
+	utilruntime.Must(apiextensionsv1.AddToScheme(options.Scheme))
+
+	setupLog := ctrl.Log.WithName("setup")
+	k8sClient, err := client.New(config.GetConfigOrDie(), options)
+	if err != nil {
+		setupLog.Error(err, "error creating client")
+		os.Exit(1)
+	}
+
+	crd := apiextensionsv1.CustomResourceDefinition{}
+	err = k8sClient.Get(context.Background(), types.NamespacedName{Name: name}, &crd)
+
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return false
+		} else {
+			setupLog.Error(err, fmt.Sprintf("Error fetching CRD for %s", name))
+			os.Exit(1)
+		}
+	}
+
+	return true
+}
```

Comment:
We have a similar function in main.go(`mapCRDAvailability`) so why can't we reuse it? 
Maybe use the util over there as well

Reply:
What about using this function in `main.go` and updating it to be similar to main.go ?

Reply:
Yeah, that's what I meant. Update the function in utils.go to be similar to main.go function. Then use that util everywhere.